docs/collecting-data/code-generation/using-the-cli/index.md

```diff
@@ -288,11 +288,21 @@ trackWebPage<Product | WebPage>(t, {
 
 </Tabs>
 
+### Data Products
+
+To generate code for the whole set of Event Specifications of a Data Product, either manually or through the `snowtype patch` command, you would need the ID of the Data Product. You can either click on the `Implement tracking` button on the Data Products main page and get the command directly:
```

Comment:
To generate code for the entire set of Event Specifications of a Data Product, either manually or through the snowtype patch command, you will need the ID of the Data Product. You can do this by either clicking the Implement tracking button on the Data Products main page to get the command directly:

Reply:
Fixed!